pinot-common/src/main/java/org/apache/pinot/common/metrics/BrokerGauge.java

```diff
@@ -26,7 +26,8 @@
  *
  */
 public enum BrokerGauge implements AbstractMetrics.Gauge {
-  QUERY_QUOTA_CAPACITY_UTILIZATION_RATE("tables", false), NETTY_CONNECTION_CONNECT_TIME_MS("nettyConnection", true);
+  QUERY_QUOTA_CAPACITY_UTILIZATION_RATE("tables", false), NETTY_CONNECTION_CONNECT_TIME_MS("nettyConnection", true),
+  TABLE_MIN_LOW_WATER_MARK("tables", false);
```

Comment:
what does this represent?

Reply:
It represents how many table pinot broker is collecting the metrics for. The exact usage of the metrics will show up in later diffs